src/hooks/useProjectEvents.ts

```diff
@@ -0,0 +1,79 @@
+import {
+  OrderDirection,
+  ProjectEvent_OrderBy,
+  ProjectEventsQuery,
+  useProjectEventsQuery,
+} from 'generated/graphql'
+import { client } from 'lib/apollo/client'
+import { PV } from 'models/pv'
+
+export type ProjectEventFilter =
+  | 'all'
+  | keyof Pick<
+      ProjectEventsQuery['projectEvents'][number],
+      | 'addToBalanceEvent'
+      | 'burnEvent'
+      | 'configureEvent'
+      | 'deployedERC20Event'
+      | 'deployETHERC20ProjectPayerEvent'
+      | 'distributePayoutsEvent'
+      | 'distributeReservedTokensEvent'
+      | 'payEvent'
+      | 'printReservesEvent'
+      | 'projectCreateEvent'
+      | 'redeemEvent'
+      | 'setFundAccessConstraintsEvent'
+      | 'tapEvent'
+      | 'v1ConfigureEvent'
+    >
+
+type ProjectEventsQueryArgs = {
+  filter?: ProjectEventFilter
+  pv?: PV
+  wallet?: string
+  projectId?: number
+  skip?: number
+  first?: number
+}
+
+export function useProjectEvents({
```

Comment:
light wrapper around the Apollo generated `useProjectEventsQuery` that simplifies arguments

Reply:
I like this a lot. I wonder if we should group all subgraph-related hooks in a directory. NBD for now.